src/components/Insight/InsightsScrollable.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Panel } from '@santiment-network/ui'
+import InsightsLatest from './InsightsLatest'
+import InsightsFeatured from './InsightsFeatured'
+import styles from './InsightsScrollable.module.scss'
+
+export const TYPES = {
+  latest: 'LATEST',
+  featured: 'FEATURED'
+}
+
+const InsightsScrollable = ({ maxLines, multilineTextId, type }) => (
+  <div className={styles.insights}>
+    <Panel className={styles.insights__panel}>
+      <div className={styles.insights__list}>
+        {type === TYPES.featured && (
+          <InsightsFeatured
+            maxLines={maxLines}
+            multilineTextId={multilineTextId}
+            className={styles.insights__card}
+          />
+        )}
+        {type === TYPES.latest && (
```

Comment:
This type of `if statements` (LOC `17`,`24`) could be refactored with a dictionary data structure  
```js
const El = TYPES[type];
<El maxLines=....... />
```